recipes/scherbelberg/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "scherbelberg" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/scherbelberg-{{ version }}.tar.gz
+  sha256: a87fb41eb666a3a5d5b0c1aa2caaa2e10a646c382bb88a4373c997dd1dc2a708
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - "scherbelberg = scherbelberg._cli:cli"
```

```suggestion
    - scherbelberg = scherbelberg._cli:cli
```
Just cosmetics. Or was that added deliberately?

---

No, just stolen from another conda-forge package that I used as a template.

---

Ahh ok. Just for future reference, if the package is python, the easiest way to generate the recipe is using [Grayskull](https://github.com/conda-incubator/grayskull). It's not always perfect but pretty close to it.